src/main/java/edu/harvard/iq/dataverse/license/reorder/LicenseReorderPage.java

```diff
@@ -0,0 +1,73 @@
+package edu.harvard.iq.dataverse.license.reorder;
+
+
+import edu.harvard.iq.dataverse.DataverseSession;
+import edu.harvard.iq.dataverse.PermissionsWrapper;
+import edu.harvard.iq.dataverse.license.License;
+import edu.harvard.iq.dataverse.license.LicenseDAO;
+import edu.harvard.iq.dataverse.license.dto.LicenseReorderDto;
+import org.apache.commons.lang.StringUtils;
+
+import javax.faces.view.ViewScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+@ViewScoped
+@Named("LicenseReorderPage")
+public class LicenseReorderPage implements Serializable {
+
+    @Inject
+    private DataverseSession session;
+
+    @Inject
+    private PermissionsWrapper permissionsWrapper;
+
+    @Inject
+    private LicenseDAO licenseDAO;
+
+    private List<LicenseReorderDto> licenses = new ArrayList<>();
+
+    // -------------------- GETTERS --------------------
+
+    public List<LicenseReorderDto> getLicenses() {
+        return licenses;
+    }
+
+    // -------------------- LOGIC --------------------
+
+    public String init() {
+
+        if (!session.getUser().isSuperuser()) {
+            return permissionsWrapper.notAuthorized();
+        }
+
+        licenses = licenseDAO.findLocalizedLicenses(new Locale(session.getLocaleCode()));
```

Comment:
If you would do Locale.forLanguageTag it would be somewhat ready for broader range of locales (for example en-US, en-GB). Single arg constructor is for language part only.

Reply:
Will we ever want to support country specific locales? So far we are just using locales from properties file.

Reply:
I don't know if we ever would want to have country specific ones. In properties there is already support for that. We could have bundle-en-US.properties and it should work out of the box.

If we would change this to forLanguageTag there would be one place less (that is rather hard to find) that needed some corrections and the cost of changing it now is rather low.